frontend/src/component/project/ProjectList/ProjectList.test.tsx

```diff
@@ -0,0 +1,37 @@
+import { render } from 'utils/testRenderer';
+import { ProjectListNew } from './ProjectList';
+import { screen, waitFor } from '@testing-library/react';
+import { testServerRoute, testServerSetup } from '../../../utils/testServer';
+import { CREATE_PROJECT } from '../../providers/AccessProvider/permissions';
+
+const server = testServerSetup();
+
+const setupApi = () => {
+    testServerRoute(server, '/api/admin/ui-config', {
```

Comment:
Cool, is this how we stub out backends for testing?

Reply:
yup, we mock HTTP with the msw library wrapped in our own abstraction (https://mswjs.io/)